ci-operator/step-registry/ipi/nutanix/ovn/hybrid/ipi-nutanix-ovn-hybrid-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: ipi-nutanix-ovn-hybrid
+  steps:
+    pre:
+    - chain: ipi-conf-nutanix
+    - ref: ovn-conf
+    - ref: ovn-conf-hybrid-manifest
+    - chain: ipi-install
+    post:
+    - chain: ipi-deprovision
```

Comment:
Take a look https://github.com/openshift/release/blob/81652219f121ff71071a9ec7d495fb8a827247c4/ci-operator/step-registry/ipi/nutanix/post/ipi-nutanix-post-chain.yaml#L4

Reply:
changed to use "ipi-nutanix-post".